rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -128,9 +128,9 @@
     private Long nextFederationCreationBlockHeight; // if -1, then clear value
     private Script lastRetiredFederationP2SHScript;
 
-    private Keccak256 fastBridgeDerivationArgumentsHashToSave = null;
-    private Sha256Hash fastBridgeBtcTxHashToSave = null;
-    private FastBridgeFederationInformation fastBridgeFederationInformationsToSave = null;
+    private Keccak256 fastBridgeDerivationArgumentsHash;
+    private Sha256Hash fastBridgeBtcTxHash;
+    private FastBridgeFederationInformation fastBridgeFederationInformations;
```

Comment:
```suggestion
    private FastBridgeFederationInformation fastBridgeFederationInformation;
```
You have a trailing `s`